devmaster/themes/boots/template.php

```diff
@@ -884,48 +885,49 @@ function boots_preprocess_node_task(&$vars) {
  * Preprocessor for Project Nodes.
  * @param $vars
  */
-function boots_preprocess_node_project(&$vars){
+function boots_preprocess_node_project(&$vars) {
   global $user;
 
   // Easy Access
   $node = &$vars['node'];
   $project = $vars['project'] = $vars['node']->project;
 
   // Live Domain link.
-  if ($project->settings->live['live_domain']) {
+  if ($project->settings->live['live_domain'] ?? NULL) {
```

Comment:
What does `??` do when in an "if" statement? I thought it was for concatenation...

Reply:
It avoids the E_NOTICE entry in the run time logs, if settings is null or the array key doesn't exist.